jake/__main__.py

```diff
@@ -197,21 +201,32 @@ def sbom(verbose, quiet, conda, targets, output):
 # ddt (ossi) subcommand
 @main.command()
 @__add_options(__shared_options)
-def ddt(verbose, quiet, conda, targets):
+@click.option(
+    '-o', '--output-format',
+    help='Specify ddt output format(json or hr=human-readable)')
+def ddt(verbose, quiet, conda, targets, requirements, output_format='hr'):
   """SPECIAL MOVE\n
   Allows you to perform scans backed by Sonatype's OSS Index
 
   Example usage:\n
       Python scan: jake ddt\n
       Conda scan: conda list | jake ddt -c\n
   """
-  __banner(quiet)
   __setup_logger(verbose)
   __check_stdin(conda)
 
+  if output_format == 'json':
+    __toggle_stdout(on=False)
```

Comment:
Not a comment on your changes, just noticing our use of `__toggle_stdout` is getting a bit hard to follow.

This change seems to work sanely in my local testing.

Reply:
Yeah :)))
I was trying to find a nice way to only print the json output and I suddenly noticed this boolean! Not a good way but it's solving the problem for now.